utm.yaml

```diff
@@ -150,14 +150,6 @@ components:
           enum:
           - RFC3339
 
-    CircleProperties:
-      required:
-      - radius
-      type: object
-      properties:
-        radius:
-          $ref: '#/components/schemas/Radius'
-
     Radius:
```

Comment:
Is there a way to force radius to be >0? Or at least add that to the description.

Reply:
Good call; done.  (OpenAPI does have the ability to express "greater than 0" programmatically)

Reply:
Ok. @BenjaminPelletier please resolve (I can't since I don't have write access).